__tests__/unit/crypto/transactions/factory.test.ts

```diff
@@ -102,14 +102,12 @@ describe("TransactionFactory", () => {
         // Old tests
         it("should match transaction id", () => {
             configManager.setFromPreset("testnet");
-            // tslint:disable-next-line: ban
-            [0, 1, 2, 3]
-                .map(type => createRandomTx(type))
-                .forEach(transaction => {
-                    const originalId = transaction.data.id;
-                    const newTransaction = TransactionFactory.fromData(transaction.data);
-                    expect(newTransaction.data.id).toEqual(originalId);
-                });
+            const randomTxs = [0, 1, 2, 3].map(type => createRandomTx(type));
+            for (const transaction of randomTxs) {
```

Comment:
This could be a simple `for` loop that counts up to 3 and creates the transaction inside the loop instead of having a map and loop.